### ISSUE TITLE
Questions about appropriate training Settings

### ISSUE THREAD

**Issue by deyang2000** [jjihwan/SV3D-fine-tune#14]
Hello, I encountered some confusion while trying to train the model:

1. First, I'd like to confirm whether all 21 frames generated for the same object can be used as individual reference frames. In other words, can I use all frames from 000-000 to 000-020 as training data for a same object?

2. I noticed in [[About finetuning time cost](https://github.com/jjihwan/SV3D-fine-tune/issues/5)](https://github.com/jjihwan/SV3D-fine-tune/issues/5) that the training time was mentioned. I'm wondering, do the "5 data points" refer to five different objects or to five views of the same object?

3. When training on 6 objects, I noticed that the training loss exploded:

![Image](https://github.com/user-attachments/assets/0d69906b-fc28-4a6f-b18e-67574d289789)

4. Regarding this image:

![Image](https://github.com/user-attachments/assets/54737816-27d2-4df2-b581-480cfcc1dbc1)  
This is the result I currently get:

![Image](https://github.com/user-attachments/assets/82acde98-204f-4f41-ad4f-928e263901d2)

The inference result from the original model is as follows:  
![Image](https://github.com/user-attachments/assets/07f1286c-7a77-4b02-9efe-b1ab59c5b94b)

I believe the issue might be due to insufficient convergence during training.

**Comment by rzhevcherkasy** [comment]
Hi, may I ask if you have figured out how to solve this problem? Thank you
